### ISSUE TITLE
1.1.17 throws AttributeError with `--disable-cache`

### ISSUE THREAD

**Issue by wayne-luminal** [pantsbuild/pex#260]
Our build system runs pex and throws the following stack trace. I've confirmed I don't see this with 1.1.16.

```
Traceback (most recent call last):
  File ".tox/package/bin/pex", line 11, in <module>
    sys.exit(main())
  File ".../lib/python3.4/site-packages/pex/bin/pex.py", line 533, in main
    options.cache_dir = make_relative_to_root(options.cache_dir)
  File ".../.tox/package/lib/python3.4/site-packages/pex/bin/pex.py", line 514, in make_relative_to_root
    return os.path.normpath(path.format(pex_root=ENV.PEX_ROOT))
AttributeError: 'list' object has no attribute 'format'
```

With the additional message

```
ERROR: InvocationError: 'pex . -o dist/component.pex -c component --disable-cache -v -i <url>'
```

**Comment by kwlzn** [comment]
@wayne-luminal thanks for the bug report and apologies for the breakage. we'll get this fixed up asap.

@digwanderlust looks like this was related to your change that just went out in 1.1.7. mind taking a look?

**Comment by kwlzn** [comment]
fwiw, it seems the use of `--disable-cache` is the bit that causes this.

**Comment by digwanderlust** [comment]
For some readon disable cache sets the cache dir to [], I'll add a test and fix this.

**Comment by digwanderlust** [comment]
Fix is up at: https://github.com/pantsbuild/pex/pull/261

**Comment by kwlzn** [comment]
this should be fixed in 1.1.8 - now available on pypi: https://pypi.python.org/pypi/pex/1.1.8

verified I can no longer reproduce the breakage using this release. closing.

**Comment by wayne-luminal** [comment]
👍  Thank you!
